src/pages/ReportDetailsPage.js

```diff
@@ -81,6 +83,7 @@ class ReportDetailsPage extends Component {
 
         if (ReportUtils.isPolicyExpenseChat(this.props.report) || ReportUtils.isChatRoom(this.props.report)) {
             this.menuItems.push({
+                key: 'settings',
```

Comment:
I think we should move these string values to CONST.js, no?

Reply:
Sure, done.